build.gradle

```diff
@@ -89,9 +89,9 @@ subprojects { subproject ->
 		hamcrestVersion = '1.3'
 		jackson2Version = '2.9.1'
 		junit4Version = '4.12'
-		junitJupiterVersion = '5.0.2'
-		junitPlatformVersion = '1.0.2'
-		junitVintageVersion = '4.12.2'
+		junitJupiterVersion = '5.1.0'
+		junitPlatformVersion = '1.1.0'
```

Comment:
Is it OK to upgrade to the `minor` version in our point release? 
Junit is not only a test dependency. The Cairo platform must provide these versions as well.

What am I missing in our backward compatibility policy?

Thanks

Reply:
Cairo gets its versions from [boot](https://github.com/spring-projects/spring-boot/blob/master/spring-boot-project/spring-boot-dependencies/pom.xml#L114) so it seems we were wrong anyway.

Reply:
Yes, it's a compile dependency for spring-rabbit-junit, but `optional` too; so I think we're ok.